modules/runtime-config/concepts/RuntimeConfig.js

```diff
@@ -0,0 +1,7 @@
+import { GeneratingConcept } from '@storefront-x/core'
+
+export default class RuntimeConfig extends GeneratingConcept {
+  get directory() {
+    return 'runtime/config'
```

Comment:
Am I right that it will look for "config" file inside of the "runtime" folder,
or the config folder inside the runtime folder? Anyway, what possibilities do we have regarding concepts and folder/file loading?
Can we look for a file in the module root folder? Or can multiple concepts use one folder as the main folder and then inside have a specific one per each concept (which I suppose is this case above)? Im sure is doable, but the question is if it makes sense and if it's a sustainable and transparent solution to use, right?

I'm asking because I would rather see runtime config definitions together with other configs for the module, or in the root of the module folder than in the deep nested folder like ./runtime/config/xxx.js|ts
Thanks

Reply:
This means that it will look at all of the files inside `runtime/config/` directory. E.g. `runtime/config/MAGENTO_URL.ts` which you would then access via `getRuntimeConfigValue('MAGENTO_URL')`.

Runtime config has special functionality compared to normal config so thats why its separated. But if you want to use runtime config in normal config (e.g. compatibility reason - changing `config/MAGENTO_URL.ts` from regular config to runtime config), you can just re-export runtime config inside config concept like this:

```typescript
my-module/config/MAGENTO_URL.ts

import getRuntimeConfigValue from '#ioc/utils/getRuntimeConfigValue'

export default getRuntimeConfigValue('MAGENTO_URL')
```

About the root directory. Concepts have ability to do whatever they want. So looking into the root directory or directory of other concepts is possible. But it's an edge-case and used very minimally (bus/emitters concept will look into bus/listeners concept directory to see if it should generate an emitter or not in case no listeners are listening).